modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+Selecting *Insert > Link…* or clicking the Link toolbar button did not, previously, execute the `mceLink` command. Equivalently, if the `link_quicklink` option was enabled, using the Ctrl+K or ⌘+K shortcut to open the Contextual Link Toolbar did not execute `mceLink` either.
+
+This was not consistent with other, related, actions. For example, clicking the Unlink toolbar button executed the `unlink` command. As well, not executing a command meant it was not possible to track the *Link…* dialog being opened, and the command could not be over-ridden with `addCommand`.
+
+With this update, `mceLink` is always executed when the Link dialog or Contextual Link Toolbar is opened.
+
+
+=== Choice menu items will now use the `menuitemradio` ARIA role to better reflect that only a single item can be active
+
+{productname} uses the https://w3.org/WAI/ARIA/[Accessible Rich Internet Applications] (ARIA) Working Group guidelines to make {productname} accessible to as wide a range of users as is possible.
+
+To this end, some {productname} menu items were assigned the `menuitemcheckbox` ARIA role. However, screen readers report this role as meaning each item in the menu can be toggled independently. This is not true: the options in these menus are mutually exclusive.
+
+The appropriate ARIA role for these objects is `menuitemradio` and, as of this release, this is the ARIA role these menu items are assigned.
+
+Consequently, screen readers will, correctly, report the options in these menus as mutally exclusive.
+
+
+[[improvements]]
+== Improvements
+
+{productname} 6.1 also includes the following improvements:
+
+=== All help dialog links to `https://tiny.cloud` now include `rel="noopener"` to avoid potential security issues
+
+`rel` is an attribute for the `<a>` element (and other elements). And setting this attribute to `rel="noopener"`  instructs the browser to load the resource referenced in the anchor tag in the same browsing context (ie, it instructs the browser to not open a new window).
+
+With this release, all {productname} help dialog links out to https://tiny.cloud[tiny.cloud] include the `rel="noopener"` attribute and value.
+
+This provides basic protection against window opener attacks.
+
+
+=== The `mceLink` command can now take the value `{ dialog: true }` to always open the link dialog
+
+The `mceLink` command can take a new value, `{ dialog: true }`.
+
+When set with this value, the *Link…* dialog will always open when `mceLink` is run.
+
+
+=== Support annotating certain block elements directly when using the editor annotator API
+
+Several new block elements can now have comments attached to them, including:
+
+* Tables of contents
+* Images
+* Images with captions
+* Code samples
+* media in iframes
+* embedded media
+* embedded pages
+
+When comments are showing, a 3px wide yellow (hex color #FFE89D) rectangle presents around any element with a comment attached.
```

Comment:
Likewise here I don't think we should be mentioning the color code, as there's 2 different shades by default and then it can change based on the skin used.

Reply:
Already removed reference to specific hex colour.